Makefile

```diff
@@ -531,6 +531,10 @@ ifdef ALCHEMY_ETHEREUM_GOERLI_TOKEN
 	NIM_PARAMS += -d:ALCHEMY_ETHEREUM_GOERLI_TOKEN:"$(ALCHEMY_ETHEREUM_GOERLI_TOKEN)"
 endif
 
+ifdef ALCHEMY_ETHEREUM_SEPOLIA_TOKEN
```

Comment:
@dlipicar this seems to be unused, but still unremoved code, don't see any need for it since we switched to [nim-confutils](https://github.com/status-im/nim-confutils).

Please remove in a new commit within this PR code from the line `506` till the line `564`.
Also if you could move `OPENSEA_API_KEY` to the new way of handling then you could remove that part from here as well.

Reply:
Opensea seems to be managed the new way already!
https://github.com/status-im/status-desktop/pull/12772/files#diff-e4c88de356005a46bf1fe7657caa45a87da077be8e702fac405ce08a70edcd1cR97

I'll just remove all of this then

Reply:
The one thing we don't seem to be doing in the new spot is inject the default values we have in the makefile

Reply:
Oh perhaps you meant the Tenor one? I'll move that to the new way

Reply:
@saledjenic see https://github.com/status-im/status-desktop/pull/12772/commits/edffc6abb2c3bdebee19ebfbfe1f314d03ddaa97

Reply:
> Oh perhaps you meant the Tenor one? I'll move that to the new way

Exactly, that's what I meant to, but wrote open sea. :) Sorry.